src/pybind/mgr/dashboard/frontend/src/app/ceph/smb/smb-cluster-form/smb-cluster-form.component.ts

```diff
@@ -43,13 +45,15 @@ export class SmbClusterFormComponent extends CdForm implements OnInit {
   hasOrchestrator: boolean;
   orchStatus$: Observable<any>;
   allClustering: string[] = [];
+  CLUSTERING = CLUSTERING;
```

Comment:
it would be nice to organize all constants together with Icons enum

Reply:
I'm sorry I do not understand this